src/components/DataTable/DataTable.scss

```diff
@@ -159,13 +165,18 @@ $breakpoint: 768px;
 }
 
 .Cell-total-footer {
-  border-top: border('divider');
+  border-top: border();
```

Comment:
I think we may still want `border('divider')` here but not 💯 . @alex-page would probably know

Reply:
Swapped back to using `border('divider')` since I was wrong to make the dividers light in dark mode